tubular/scripts/helpers.py

```diff
@@ -172,6 +173,8 @@ def _setup_all_apis_or_exit(fail_func, fail_code, config):
         hubspot_aws_region = config.get('hubspot_aws_region', None)
         hubspot_from_address = config.get('hubspot_from_address', None)
         hubspot_alert_email = config.get('hubspot_alert_email', None)
+        panel_backend_base_url = config['base_urls'].get('panel_backend', None)
```

Comment:
rename these variables as well.

Reply:
panel_backend added to differentiate with other services.